base_location_geonames_import/models/res_country.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Franco Tampieri, Freelancer http://franco.tampieri.info
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCountryState(models.Model):
+
+    _inherit = 'res.country'
+
+    geonames_state_name_column = fields.Integer(
+        'Geoname State Name Column',
+        default=3,
```

Comment:
One last thing: please don't put defaults on these columns, and let the keyword argument to act

Reply:
@pedrobaeza ok I fix the default and push